rpcserver.go

```diff
@@ -3058,17 +3063,24 @@ func marshalLeafKey(leafKey universe.LeafKey) *unirpc.AssetKey {
 // Taproot Asset commitment, and script_key is the script_key of the asset
 // within the Taproot Asset commitment for the given asset_id or group_key.
 func (r *rpcServer) AssetLeafKeys(ctx context.Context,
-	req *unirpc.ID) (*unirpc.AssetLeafKeyResponse, error) {
+	req *unirpc.AssetLeafKeysRequest) (*unirpc.AssetLeafKeyResponse, error) {
 
-	universeID, err := UnmarshalUniID(req)
+	universeID, err := UnmarshalUniID(req.Id)
```

Comment:
Yeah there's a tracking issue for that, I think for some it's more acceptable rn since other funcs called in the RPC handler will deal with it, or we don't access struct members directly.